src/test/datascience/reactHelpers.ts

```diff
@@ -414,7 +414,13 @@ export function setUpDomEnvironment() {
             content += 'export function getCSSBasedConfiguration() { return CSSBasedConfiguration.INSTANCE; };\n';
             mod._compile(content, filename);
         } else {
-            _oldLoader(mod, filename);
+            try {
+                _oldLoader(mod, filename);
+            } catch (e) {
```

Comment:
The new LS Packages have issues when loading them to the DOM for tests.
They're not needed so I let them fail. Let me know if we want them loaded for that.

Reply:
Hmm, what if something else fails? can we instead check if `e` is something related to LS package and then ignore it, and if its something else we re-throw the error.